cli/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ _Released 03/14/2023 (PENDING)_
 
 - It is now possible to control the number of connection attempts to the browser using the CYPRESS_CONNECT_RETRY_THRESHOLD Environment Variable. Learn more [here](https://docs.cypress.io/guides/references/advanced-installation#Environment-variables). Addressed in [#25848](https://github.com/cypress-io/cypress/pull/25848).
 
+**Bugfixes:**
+
+- Fixed an issue where using `Cypress.require()` would throw the error `Cannot find module 'typescript'`. Fixes [#25885](https://github.com/cypress-io/cypress/issues/25885).
```

Comment:
Is this fixing a regression or did it never work?

Reply:
It worked with regular `require`, but hasn't worked with `Cypress.require()` since it was re-released.